content/doc/developer/plugin-development/choosing-jenkins-baseline.adoc

```diff
@@ -29,7 +33,7 @@ When updating the core dependency, choose the link:/changelog-stable/[newest LTS
 
 ## Currently recommended versions
 
-At the moment, the Jenkins releases *PLACEHOLDER_OLDER_LTS_POINT_ONE* and *PLACEHOLDER_NEWER_LTS_POINT_ONE* make good core dependencies unless there are specific reasons, like new features, to choose a different release.
+At the moment, the Jenkins releases *PLACEHOLDER_OLDER_LTS_POINT_HIGHS, and PLACEHOLDER_NEWER_LTS_POINT_ONE* make good core dependencies unless there are specific reasons, like new features, to choose a different release.
```

Comment:
I'd keep the previous recommendations largely intact. Note that my previous feedback was more around explicitly listing _feasible_ releases within the guidelines (since it's no longer as easy as just putting ".1", there could be .3 or .4 or even .5), rather than extending the list of _recommended_ releases.

The live site recommends 2.303.1 and 2.319.1, absent reasons to choose a different baseline.

Recommending 2.303.3 and 2.319.3 instead seems a reasonable outcome of this discussion.

I think we could make the previous algorithm simpler by always recommending either of the newest two historical lines independent of recent plugin splits, but just listing 5 versions doesn't seem too helpful to users who want something more specific.

Reply:
> The Ruby code was too complex to follow also.

The idea of that algorithm is to generally recommend the .1 of the two most recent "historical" LTS lines. An exception was made if there's recently been a plugin detached from core. In that case, if there are at least two LTS.1 releases since the split (one historical, one current), recommend those instead, as that would reduce the number of implicit dependencies.

The possible simplification would be to just always recommend the last releases of the two most recent historical LTS lines.

> I was attempting to follow your suggestion in https://github.com/jenkins-infra/jenkins.io/pull/4876#issuecomment-1122708926 but apparently misunderstood.

What I meant to say in that comment was that we should be explicit about what specific point releases match the criteria laid out on that page. Those are a superset of the (fewer) versions we would recommend, absent specific reasons to choose something else. Something like that:

* 2.332.1 (current)
* 2.319.3 (historical, generally recommended)
* 2.303.3 (historical, generally recommended)
* 2.289.3 (historical)
* 2.277.4 (historical)